### DISCUSSION TITLE
Upcoming OpenDDS Version 3.28.0

### DISCUSSION THREAD

**Discussion by iguessthislldo** [OpenDDS/OpenDDS#4575] [Announcements]
The OpenDDS Foundation plans to release the master branch as OpenDDS 3.28.0 in the near future. The current release notes can be found [here](https://opendds.readthedocs.io/en/master/news.html#v3-28-0). Any feedback would be greatly appreciated.

**Comment by jwillemsen** [comment]
AXCIOMA works with the current OpenDDS master using ACE 7.1.4 and TAO 3.1.4 on RHEL 8.0 (see https://www.axcioma.org/scoreboard)

**Comment by vermaete** [comment]
If possible it would be nice to have this one solved (https://github.com/OpenDDS/OpenDDS/discussions/4552) in the upcoming release.
the security tests are passing on Ubuntu 22.04, but will probably fail on 24.04 (LTS).  As they do on the latest release of Debian and Yocto.

**Reply by jrw972** [reply]
I agree but #4552 could push the release back by weeks.  Can we do a patch release when #4552 is addressed?

**Reply by vermaete** [reply]
sure, not an issue at all.
br,
